crt_portal/cts_forms/tests/integration/report_submission.py

```diff
@@ -0,0 +1,142 @@
+    page.click("ul[role=\"presentation\"]")
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Step 2: Primary concern - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Check NOT hatecrime
+    page.check("//li[normalize-space(.)='No']/input[normalize-space(@type)='radio' and normalize-space(@name)='2-hate_crime']")
+
+    # Click //label[normalize-space(.)='No']
+    page.click("//label[normalize-space(.)='No']")
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Step 3: Location - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Click input[name="3-location_name"]
+    page.click("input[name=\"3-location_name\"]")
+
+    # Fill input[name="3-location_name"]
+    page.fill("input[name=\"3-location_name\"]", "Test store")
+
+    # Click input[name="3-location_city_town"]
+    page.click("input[name=\"3-location_city_town\"]")
+
+    # Fill input[name="3-location_city_town"]
+    page.fill("input[name=\"3-location_city_town\"]", "Testing")
+
+    # Select Alabama
+    page.selectOption('select#id_3-location_state', 'AL')
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Step 4: Personal characteristics - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Check AGE
+    page.check("#id_9-protected_class_0")
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Step 5: Date - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Click input[name="10-last_incident_month"]
+    page.click("input[name=\"10-last_incident_month\"]")
+
+    # Fill input[name="10-last_incident_month"]
+    page.fill("input[name=\"10-last_incident_month\"]", "1")
+
+    # Press Tab
+    page.press("input[name=\"10-last_incident_month\"]", "Tab")
+
+    # Press Tab
+    page.press("input[name=\"10-last_incident_day\"]", "Tab")
+
+    # Fill input[name="10-last_incident_year"]
+    page.fill("input[name=\"10-last_incident_year\"]", "2020")
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Step 6: Personal description - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Click textarea[name="11-violation_summary"]
+    page.click("textarea[name=\"11-violation_summary\"]")
+
+    # Fill textarea[name="11-violation_summary"]
+    page.fill("textarea[name=\"11-violation_summary\"]", "Test report submission")
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Step 7: Review - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Click input[type="submit"]
+    page.click("input[type=\"submit\"]")
+    assert page.title() == "Contact the Civil Rights Division | Department of Justice"  # nosec
```

Comment:
Shouldn't we test that the values we input actually show up in the confirmation page?

Reply:
Yes! If you're amenable to it I'd love to extend this test and add others as separate efforts/cards building on this approach. Ideally pairing so we can all build a bit of experience.

Reply:
Absolutely. thanks for the clarifications!